docs/getting-started-guides/photon-controller.md

```diff
@@ -0,0 +1,232 @@
+---
+---
+
+The example below creates a Kubernetes cluster using VMware's Photon
+Controller. The cluster will have one Kubernetes master and three
+Kubernetes nodes.
+
+* TOC
+{:toc}
+
+### Prerequisites
+
+1. You need administrator access to a [VMware Photon
```

Comment:
How do I get this?  Is this outside the scope of the doc?  I am trying to test that performing the steps in this doc will get me a cluster with kbuectl setup, but don't know enough about Photo to set it up.